sbin/start_l1.sh

```diff
@@ -109,6 +112,10 @@ if [ "$L1_STACK" = "geth" ]; then
     $L1_GETH_BIN attach --exec \
       "eth.sendTransaction({ from: eth.coinbase, to: '"$DEPLOYER_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
       $L1_ENDPOINT
+
+    # Wait for 1 block
+    echo "Waiting for chain progression..."
+    sleep 4
 elif [ "$L1_STACK" = "hardhat" ]; then
     echo "Using $CONTRACTS_DIR as HH proj"
     cd $CONTRACTS_DIR && npx hardhat node --no-deploy --hostname $L1_HOST --port $L1_PORT &> $LOG_FILE &
```

Comment:
I think we should set the HH ID to 1337 here to avoid confusion

Reply:
will do in next one